code/modules/libvg/utf8.dm

```diff
@@ -21,6 +21,8 @@
 
 // Converts a byte string to a UTF-8 string, sanitizes it and caps the length.
 /proc/utf8_sanitize(var/message, var/mob_or_client, var/length)
+	if (!length)
+		length = min(length(message), MAX_MESSAGE_LEN)
```

Comment:
I don't know about this, maybe you should use `utf8_len`. And at that point it should probably be handled by libvg, so...
paging @PJB3005